modules/console/nixos.nix

```diff
@@ -8,20 +8,20 @@ with config.lib.stylix.colors;
 
   config.console.colors = lib.mkIf config.stylix.targets.console.enable [
     base00-hex
-    base08-hex
-    base0B-hex
-    base0A-hex
-    base0D-hex
-    base0E-hex
-    base0C-hex
-    base05-hex
-    base03-hex
-    base09-hex
-    base01-hex
-    base02-hex
-    base04-hex
-    base06-hex
-    base0F-hex
+    red
+    green
+    yellow
+    blue
+    magenta
+    cyan
+    base07-hex
+    base00-hex
```

Comment:
https://github.com/philj56/base16-linux-vt/blob/221e64fad4a5675f96d2d3492fdaabfb59e05335/templates/default.mustache#L5-L20 uses `base03` here. Perhaps some apps try using it as a text color.